### DISCUSSION TITLE
Display room next to entity names

### DISCUSSION THREAD

**Discussion by innocenzi** [cgiesche/streamdeck-homeassistant#244] [Feature Requests]
It can be hard to know which entity we are working with when their name is the same. In the Home Assistant UIs, the room name context is given every time, but it's not in the Stream Deck plugin:

![CleanShot 2024-02-06 at 04 11 34](https://github.com/cgiesche/streamdeck-homeassistant/assets/16060559/0b2f673a-0c34-4da7-9374-5d235aeb17ff)

**Comment by cgiesche** [comment]
Hi,

i do not currently get any room information from home assistant. As a first step, I added an option to search entities by their name OR entityId (https://github.com/cgiesche/streamdeck-homeassistant/discussions/248#discussioncomment-8440643). This is not yet a final solution, but better than the current state. It will be released during march.
Besides from that, I will review the entity selection component soon.

**Reply by innocenzi** [reply]
Thank you!
